opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -152,3 +154,57 @@ def _python_path_without_directory(python_path, directory, path_separator):
         "",
         python_path,
     )
+
+
+_OTEL_SEMCONV_STABILITY_OPT_IN_KEY = "OTEL_SEMCONV_STABILITY_OPT_IN"
+
+
+class _OpenTelemetryStabilitySignalType:
+    HTTP = "http"
+
+
+class _OpenTelemetryStabilityMode:
+    # http - emit the new, stable HTTP and networking conventions ONLY
+    HTTP = "http"
+    # http/dup - emit both the old and the stable HTTP and networking conventions
+    HTTP_DUP = "http/dup"
+    # default - continue emitting old experimental HTTP and networking conventions
+    DEFAULT = "default"
+
+
+class _OpenTelemetrySemanticConventionStability:
```

Comment:
Let's add tests in this package so the burden isn't on each instrumentation that will use this.

Reply:
I didn't add tests yet because I wanted to see what people thought about the approach. If we get a general concensus I will start adding tests.